dbcon/joblist/jlf_tuplejoblist.cpp

```diff
@@ -319,7 +321,7 @@ void updateExp2Cols(JobStepVector& expSteps, TableInfoMap& tableInfoMap, JobInfo
   }
 }
 
-void adjustLastStep(JobStepVector& querySteps, DeliveredTableMap& deliverySteps, JobInfo& jobInfo)
+void adjustLastStep(JobStepVector& querySteps, DeliveredTableMap& deliverySteps, JobInfo& jobInfo, const uint32_t orderby_threads)
```

Comment:
jobInfo.orderByThreads has this info so you don't need to change the API to propagate the value.

Reply:
Removed.